### PR TITLE
Update GCP Windows node image versions

### PR DESCRIPTION
**Pull request by pjh**
- Makes Windows Server 2019 the default version for Windows clusters on
  GCP, since 1809 will be EOL in a few months.
- Adds Windows Server version 1909 as a Windows node choice.
- Use Windows images with updates from January 2020.
- Cleans up the code that sets the node image.

/kind cleanup

```release-note
NONE
```

### CODE REVIEW

**Comment by pjh** [comment]
The `WINDOWS_NODE_OS_DISTRIBUTION` selector is working as expected. When I run:

```
$ NUM_NODES=2 NUM_WINDOWS_NODES=2 KUBE_GCE_ENABLE_IP_ALIASES=true KUBERNETES_NODE_PLATFORM=windows LOGGING_STACKDRIVER_RESOURCE_TYPES=new WINDOWS_NODE_OS_DISTRIBUTION=win1909 KUBERNETES_SKIP_CONFIRM=y ./cluster/kube-up.sh
```

I see this in the Windows node's serial port output:

```
2020/01/28 22:32:55 windows-startup-script-ps1: GCE Windows image:
2020/01/28 22:32:55 windows-startup-script-ps1: projects/windows-cloud/global/images/windows-server-1909-dc-core-for-containers-v20200114
```

**Comment by pjh** [comment]
I glanced at these test results, none of them seem relevant to this PR.

pull-kubernetes-e2e-windows-gce is non-blocking.

> Test name	Commit	Details	Rerun command
> pull-kubernetes-e2e-windows-gce	[fad960b](https://github.com/kubernetes/kubernetes/commit/fad960bad1583ecde678c2387cd4f251a7df60ba)	[link](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87629/pull-kubernetes-e2e-windows-gce/1222329796126576640)	`/test pull-kubernetes-e2e-windows-gce`
> pull-kubernetes-e2e-gce-device-plugin-gpu	[8f76b7e](https://github.com/kubernetes/kubernetes/commit/8f76b7e0b936534625202b5c164401d3047f8791)	[link](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87629/pull-kubernetes-e2e-gce-device-plugin-gpu/1222478274655948800/)	`/test pull-kubernetes-e2e-gce-device-plugin-gpu`
> pull-kubernetes-e2e-gce	[8f76b7e](https://github.com/kubernetes/kubernetes/commit/8f76b7e0b936534625202b5c164401d3047f8791)	[link](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87629/pull-kubernetes-e2e-gce/1222478274643365889/)	`/test pull-kubernetes-e2e-gce`

**Comment by pjh** [comment]
Rebased to merge edits in a markdown file, now have to struggle to get tests to pass again :'(

File: cluster/gce/windows/README-GCE-Windows-kube-up.md [hunk 1]
```diff
@@ -95,21 +98,24 @@ To teardown the cluster run:
...
+```
+PROJECT=${CLOUDSDK_CORE_PROJECT} WINDOWS_NODE_OS_DISTRIBUTION=win2019 \
+  ./hack/e2e-internal/e2e-up.sh
+```
 
-If any e2e cluster exists already, this command will prompt you whether tears down and creates a new	one. To teardown existing e2e cluster only, run the command:
-```	
-PROJECT=${CLOUDSDK_CORE_PROJECT} ./hack/e2e-internal/e2e-down.sh	
-```	
+If any e2e cluster exists already, this command will prompt you whether tears
```
Comment (mtaufen) [inline cluster/gce/windows/README-GCE-Windows-kube-up.md]: s/whether tears down and creates/to tear down and create

Comment (pjh) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md]: Fixed.

**Pull request merged**